src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -58,7 +59,7 @@
     private final Duration gracefulShutdownQuietPeriod;
     private final Duration gracefulShutdownTimeout;
 
-    private final Executor blockingTaskExecutor;
+    private final ExecutorService blockingTaskExecutor;
```

Comment:
how about renaming field into `blockingTaskExecutorService`? it might be breaking, but less confusing.